src/styles/blocks/header.scss

```
@@ -0,0 +1,70 @@
+.header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  background-color: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
```

use nesting, fix in all cases
```suggestion
  &__logo {
```